.generator/src/generator/templates/model_utils.j2

```
@@ -1042,11 +1048,18 @@ def deserialize_primitive(data, klass, path_to_item):
                     raise ValueError("This is not a date")
                 return parse(data).date()
         else:
-            converted_value = klass(data)
+            if isinstance(data, str) and klass == UUID:
+                try:
+                    converted_value = UUID(data)
+                except ValueError:
+                    print("This is not an UUID")
```

Need to remove that print. Should we let the error be raised?